src/python/grpcio/grpc/aio/_base_server.py

```diff
@@ -211,6 +211,22 @@ def invocation_metadata(self) -> Optional[Metadata]:
           The invocation :term:`metadata`.
         """
 
+    @abc.abstractmethod
+    def is_active(self) -> bool:
+        """ Checks whether the context is active
+
+        Returns:
+          boolean conveying whether the context is active
+        """
```

Comment:
```
        """Describes whether the RPC is active or has terminated.

        Returns:
          True if RPC is active, False otherwise.
        """
```